src/main/java/frc/robot/subsystems/Shooter.java

```diff
@@ -0,0 +1,202 @@
+package frc.robot.subsystems;
+
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.utilities.Functions;
+import frc.robot.utilities.lists.Ports;
+
+/**
+ * Subsystem for the Shooter mechanism.
+ */
+public class Shooter extends SubsystemBase {
+
+    public static final double
+            P = 0,
+            I = 0,
+            D = 0,
+            FF = 0,
+            IZ = 0,
+            MAX_RPM = 0;
+
+    private final CANSparkMax shooterMotor = new CANSparkMax(
+            Ports.SHOOTER_MOTOR,
+            CANSparkMaxLowLevel.MotorType.kBrushless);
+
+    private final DoubleSolenoid hood = new DoubleSolenoid(
```

Comment:
we are useing a single solenoid, it still gives us 2 positions (true and false), and is much cheaper. the downside and whi we dident use them in the passed is that they dont rember what state they are in when they loose power and always deafult to false